src/django/api/admin.py

```diff
@@ -61,6 +61,18 @@ class FacilityHistoryAdmin(SimpleHistoryAdmin):
     readonly_fields = ('created_from',)
 
 
+class FacilityListAdmin(admin.ModelAdmin):
+    readonly_fields = ('replaced_by_link',)
```

Comment:
If this is just `replaced_by`, Django will automatically create a link to the replacing facility list without the method below. However, it displays a label of `Facility list:` and I couldn't figure out how to override that.